installers/windows/minikube.nsi

```diff
@@ -0,0 +1,176 @@
+# This installs two files, minikube.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
+# adds uninstall information to the registry for Add/Remove Programs
+ 
+# To get started, put this script into a folder with the two files (minikube.exe, logo.ico, and license.rtf -
+# You'll have to create these yourself) and run makensis on it
+ 
+# If you change the names "minikube.exe", "logo.ico", or "license.rtf" you should do a search and replace - they
+# show up in a few places.
+# All the other settings can be tweaked by editing the !defines at the top of this script
+!define APPNAME "Minikube"
+!define COMPANYNAME "Kubernetes"
+!define DESCRIPTION "A Local Kubernetes Development Environment"
+# These three must be integers
+!define VERSIONMAJOR --VERSION_MAJOR--
+!define VERSIONMINOR --VERSION_MINOR--
+!define VERSIONBUILD --VERSION_BUILD--
+# These will be displayed by the "Click here for support information" link in "Add/Remove Programs"
+# It is possible to use "mailto:" links in here to open the email client
+!define HELPURL "https://github.com/kubernetes/minikube" # "Support Information" link
+!define UPDATEURL "https://github.com/kubernetes/minikube/releases" # "Product Updates" link
+!define ABOUTURL "https://github.com/kubernetes/minikube" # "Publisher" link
+# This is the size (in kB) of all the files copied into "Program Files"
+!define INSTALLSIZE --INSTALL_SIZE--
+ 
+RequestExecutionLevel admin ;Require admin rights on NT6+ (When UAC is turned on)
+ 
+InstallDir "$PROGRAMFILES\${COMPANYNAME}\${APPNAME}"
+!define UNINSTALLDIR "Software\Microsoft\Windows\CurrentVersion\Uninstall\${COMPANYNAME} ${APPNAME}"
+ 
+# rtf or txt file - remember if it is txt, it must be in the DOS text format (\r\n)
+LicenseData "LICENSE.txt"
+# This will be in the installer/uninstaller's title bar
+Name "${COMPANYNAME} - ${APPNAME}"
+Icon "logo.ico"
+outFile "minikube-installer.exe"
+ 
+!include "LogicLib.nsh"
+!include "EnvVarUpdate.nsh"
+!include "MUI.nsh"       ; Modern UI
+ 
+ 
+# !insertmacro MUI_PAGE_WELCOME

```

Comment:
I would suggest adding:
MUI_ICON and MUI_UNICON: Icon for the installer and uninstaller
MUI_WELCOMEFINISHPAGE_BITMAP and MUI_UNWELCOMEFINISHPAGE_BITMAP: Gives a nice branded feel to the installer first page
MUI_HEADERIMAGE_BITMAP:  Gives a nice branded feel to the installers other pages (hanldes uninstaller too)